subjects/localhost/README.md

```diff
@@ -0,0 +1,78 @@
+## Localhost
+
+Finally you are going to understand how internet works from the server side. The Hypertext Transfer Protocol was created in order to ensure a reliable way to communicate on a request/response base.
+
+This protocol is used by servers and clients (usually browsers) to serve content and it is the backbone of the World Wide Web, still it is also used in many other cases that are far beyond the scope of this exercise.
+
+Here you will learn the basics of the protocol and a good place to start could be the [HTTP/1.1 RFC](https://www.rfc-editor.org/rfc/rfc9112.html).
+
+
+### Instructions
+
+- The project must be written in `Rust`.
+- You can to use `libc` crate for functions as `epoll` or equivalents and any other necessary system call that Rust do not implement natively.
+- You may need to use `unsafe` keyword for some operations (don't abuse it).
+
+> You can't use crates that already implement server features like `tokio` or `nix`.
+
+#### The Server
```

Comment:
In my opinion it is better for the student to check what a server is... since it can be may thinks and different students may have different interest (so one will read 1000 lines articles about it and others only the first 3 lines from wikipedia)... if we quote wikipedia definition then everybody will just stand by it.
let me know if it works for you, otherwise we put a small quote.

Reply:
You're right, what i was finding very odd was the fact that nothing pointed out the goal of the proj so i thought  adding something to direct them to building a server was necessary here, but the line you just added is enough